decomposed/extensions/MDDE_Model_Export_Extension/MDDE_Model_Export_Extension.xem

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?PowerDesigner AppLocale="UTF16" AutoAttach="FALSE" Code="MDDE_Model_Export_Extension" HasExtendedMetaClass="FALSE" ID="{0297A8FE-93C4-4612-A9E0-CD7292BFF652}" LibID="{26BBDCC2-15B0-4F8B-9176-3E9D3A44E3CC}" Name="Model Driven Data Engineering Export Extension" Type="{186C8AC3-D3DC-11D3-881C-00508B03C75C}" signature="XEM_XML" version="16.6.10.6261"?>
+<!-- do not edit this file -->
+
+<EXASET xmlns:a="attribute" xmlns:c="collection" xmlns:o="object">
+
+<o:ExtendedModelDefinition Id="0297A8FE-93C4-4612-A9E0-CD7292BFF652">
+<a:ObjectID>0297A8FE-93C4-4612-A9E0-CD7292BFF652</a:ObjectID>
+<a:Name>Model Driven Data Engineering Export Extension</a:Name>
```

Comment:
Voor voorstel is om hier "Model Driven Data Engineering - Export Extension (MDDE)" van te maken.

Reply:
ok

Reply:
@harmen-xb verwerkt